tests/integration/it_employees_test.py

```diff
@@ -34,7 +41,9 @@ def test_timestamp_mixin_created_on(db_session):
 def test_timestamp_mixin_updated_on(db_session):
     """Integration test for testing TimestampsMixin updated_on field"""
     before = datetime.utcnow()
-    employee = create_employee()
+    company = create_company()
```

Comment:
@VladimirRyzhkov Take a look at this one pls
https://github.com/timelesslounge/timelessis/pull/395/files#diff-882d8d4ba8a04d60cd877077a65f60f7R79

Reply:
@vladarefiev Vlad, I fixed. Please look again.